pkg/api/series.go

```diff
@@ -71,12 +71,12 @@ func series(queryable *query.Queryable) http.HandlerFunc {
 		var sets []storage.SeriesSet
 		var warnings storage.Warnings
 		for _, mset := range matcherSets {
-			s, _, wrn, err := q.Select(false, nil, nil, mset...)
-			warnings = append(warnings, wrn...)
-			if err != nil {
-				respondError(w, http.StatusUnprocessableEntity, err, "execution")
-				return
-			}
+			s, _ := q.Select(false, nil, nil, mset...)
+			//warnings = append(warnings, wrn...)
+			//if err != nil {
```

Comment:
We should probably check s.Err() here? and not just comment out?

Reply:
This is addressed. Hence, resolving.